data/scripts.js

```diff
@@ -140,7 +140,10 @@ exports.BattleScripts = {
 				this.add('-notarget');
 				return true;
 			}
-			if (targets.length > 1) move.spreadHit = true;
+			if (targets.length > 1) {
+				this.attrLastMove('[spread]');
+				move.spreadHit = true;
+			}
```

Comment:
Even better, though, would be to move this below, and send the list of successful targets in `[spread]`